scripts/qa_crowbarsetup.sh

```diff
@@ -4443,6 +4454,42 @@ function onadmin_run_cct()
     return $ret
 }
 
+function onadmin_devsetup()
+{
+    # install dev setup dependencies
+    add_sdk_repo
+    ensure_packages_installed git gcc ruby2.1-devel sqlite3-devel libxml2-devel
+
+    # create development folders
+    mkdir -p /opt/crowbar/crowbar_framework/db /opt/crowbar/barclamps
+
+    # copy existing database
+    cp /opt/dell/crowbar_framework/db/production.sqlite3 /opt/crowbar/crowbar_framework/db/development.sqlite3
+
+    # clone git repos
+    local crowbar_git_dir=/opt/crowbar/git/crowbar
+    git clone https://github.com/crowbar/crowbar.git $crowbar_git_dir
+    git clone https://github.com/crowbar/crowbar-core.git $crowbar_git_dir/barclamps/core
+    git clone https://github.com/crowbar/crowbar-openstack.git $crowbar_git_dir/barclamps/openstack
+    git clone https://github.com/crowbar/crowbar-ceph.git $crowbar_git_dir/barclamps/ceph
+    git clone https://github.com/crowbar/crowbar-ha.git $crowbar_git_dir/crowbar/barclamps/ha
+
+    # install development gems and generate dir tree
+    pushd $crowbar_git_dir
+    bundle install
+    GUARD_SYNC_HOST=localhost bundle exec guard
```

Comment:
again, using guard from my host is really preferable. and this is just a one shot, and then you call it again in the foreground when the cloud is installed? it's not really visible or documented from here like in https://github.com/SUSE-Cloud/automation/pull/712/files#diff-4da4ef75e4826a188f831da36089295cR1095